File: tests/fixtures/v2-outbound-flow/positive/outbound_flow.py
def direct(request):
    return requests.get(request.args["direct"])


def propagated(request):
    requested = request.args["propagated"]
    alias = requested
    return requests.get(alias)


def parsed(request):
    destination = urllib.parse.urlsplit(request.args["parsed"])
    return requests.get(destination.geturl())


def valid_scheme(destination):
    return destination.scheme == "https"

File: tests/fixtures/v2-outbound-flow/positive/outbound_flow.go
package outboundflow

func direct(r *Request) {
	_, _ = http.Get(r.FormValue("direct"))
}

func propagated(r *Request) {
	requested := r.FormValue("propagated")
	alias := requested
	_, _ = http.Get(alias)
}

func parsed(r *Request) {
	destination, _ := url.Parse(r.FormValue("parsed"))
	_, _ = http.Get(destination.String())
}

func validScheme(destination *url.URL) bool {
	return destination.Scheme == "https"
}
